docs/api.md

```diff
@@ -4608,6 +4609,7 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 + isNew: `true` (boolean, required) - Whether or not this Property is new (will be created by publishing the Draft Form).
 
 ## Property Detailed (object)
-+ name: `first_name` (string, required) - The name of the Property.
++ name: `the.age` (string, required) - The name of the Property.
++ odataName: `the_age` (string, required) - OData santized name of the property.
```

Comment:
I'm thinking that a little more explanation might be helpful:

> The name of the property as it will appear in OData. OData property names can only contain alphanumeric characters and underscores.